.github/workflows/codeql.yml

```diff
@@ -73,6 +73,8 @@ jobs:
       env:
         GITHUB_TOKEN: ${{secrets.KIBANAMACHINE_TOKEN}}
         SLACK_TOKEN: ${{secrets.CODE_SCANNING_SLACK_TOKEN}}
+        CODEQL_ES_HOST: ${{secrets.CODE_SCANNING_ES_HOST}}
```

Comment:
> [!TIP]
> It might feel a bit more consistent if we use the same prefix

```suggestion
        CODE_SCANNING_ES_HOST: ${{secrets.CODE_SCANNING_ES_HOST}}
```